### DISCUSSION TITLE
「Low code / No code」工具的 AI 進化論

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS3073-C/Python-Community-GS3073-C#659] [General]
根據 IT 市調公司「IDC Japan」，調查日本境內的企業與公部門等共 500 個機關，導入 Low code 等的比例，2021 年是 2020 年的 4.4 倍，已達到 37.4%，而且估計 2023 年新開發的 App 當中將會有 6 成是 Low code 等所開發的。

日本越來越多企業積極導入即使沒有程式背景的人也可以用更直覺的方式開發所需的「Low code / No code」工具，此文中有提到很多公司的應用例子，歡迎大家點進去參考。

不過我剛開始看到這個標題的時候覺得很矛盾，現今的社會不是人人都要會寫程式嗎，應該是要有更多產業跟工程師的寫code能力結合才對啊！後來我繼續看到本文最後，才發現原來是數位轉型人才供應不足！日經分析，企業內部自行製作軟體的動向開始擴大，主因是相對於企業推行數位轉型的需求、工程師的培養卻完全追不上所需。日本經濟產業省預測，到了 2030 年日本將出現最多 79 萬人的資訊人力缺口。
我認為大家可以好好思考怎麼應用這個人才缺口，除了增進自己寫code的能力是不是也能協助企業設計更多方便使用的軟體補足這個缺口！

文章連結：https://www.inside.com.tw/article/30251-AI%20DX%20LowCode%20NoCode%20tools%20Japan

**Comment by 710388** [comment]
這篇文章非常有趣，但是程式設計需求增加 也可以試著進入相關領一樣能發財

**Comment by bww0401** [comment]
感覺現在會打程式已經不能成為優勢了，我們應該積極培養更多專長以及創造力，以成為不容易被替代的人力!

**Comment by 0212Jerry** [comment]
數位轉型人才不足的問題在我看來只是短期內的問題，在Ai 的概念更廣為流傳後我相信會有更多人才出現，而能做的事情除了增進自己的實力之外，同時發展另一個領域，讓自己成為一個多功能的人才，才不容易被取代。
